hooks/post/34-pulpcore_directory_layout.rb

```diff
@@ -0,0 +1,8 @@
+unless app_value(:noop)
+  require 'pathname'
+
+  LEGACY_DIR = Pathname.new('/var/lib/pulp/artifact')
+
+  logger.debug("Removing legacy symlink #{LEGACY_DIR}")
+  LEGACY_DIR.unlink if LEGACY_DIR.symlink?
+end
```

Comment:
I am not asking for implementation here. But I think this provides a good opportunity to ask a question regrading noop. If we were aiming for true noop and reporting of what *would* change. Should we look in our hooks to present information about what would have been done? For example, detecting that it is a symlink and informing the user this would unlink it.

Reply:
I think that's a fair question and I would actually consider it in this PR. Abusing noop in tests and checking logs is actually often a decent way to test code paths.